nym-vpn-core/crates/nym-vpn-account-controller/tests/integrations/common/mod.rs

```diff
@@ -60,12 +60,15 @@ pub fn mock_user_agent() -> nym_http_api_client::UserAgent {
     }
 }
 
-pub fn mock_mnemonic() -> Mnemonic {
-    Mnemonic::parse::<&str>("dash hungry rate famous lesson march suit refuse excite soul faith bid buddy tortoise melody advice dirt coffee fluid sure air decrease cargo work").unwrap()
+pub fn mock_account() -> StorableAccount {
```

Comment:
Could we add a quick test with a decentralised account?
Setup with No API route, storing a decentralised account and asserting ReadyState?
Maybe throw a little offline/online in the mix in there somewhere?